### DISCUSSION TITLE
SysEx not working

### DISCUSSION THREAD

**Discussion by batata004** [djipco/webmidi#445] [General]
Hi,

This library is awesome and I managed to make everything I need to work, except SysEx. I have an Arturia midi controller and I got in touch with then to know how to send the command OCTAVE UP. They told me the SysEx message should be:

    `F0 00 20 6B 7F 42 02 00 00 11 7F F7`

Then I used webmidi library to send the message. I used this simple code below, extracting the device ID and data from the command above (provided by the manufacturer itself).

    WebMidi.outputs[0].sendSysex([0x00, 0x20, 0x6B],[0x7F,0x42,0x02,0x00,0x00,0x11,0x7F]);

But nothing happens after executing this code. I used `{sysex:true}` and got no errors, everything looks fine. But the SysEx message looks not being understood or sent. Any other messages (notes, command change...) work pretty well with webmidi library.

**Comment by djipco** [comment]
As far as I can tell, WEBMIDI.js is behaving normally. I tried your code and directed the output to MIDI Monitor (a Mac app to troubleshoot MIDI communication). As you can see, the output is as expected

<img width="799" alt="Capture d’écran, le 2024-10-17 à 17 52 57" src="https://github.com/user-attachments/assets/b260c9fa-3bdc-446f-9d10-ce5069182645">

I'm tempted to say that the problem is either with your device or with the command itself. Let me know if I can help any further. In the meantime, I'm going to move this issue to the discussion section.
